src/helm/impress/values.yaml

```
@@ -234,10 +240,17 @@ backend:
  ## @param backend.migrate.restartPolicy backend migrate job restart policy
   migrate:
     command:
-      - "python"
-      - "manage.py"
-      - "migrate"
-      - "--no-input"
+      - /bin/sh
+      - "-c"
+      - |
+        while ! python manage.py check --database default > /dev/null 2>&1
+        do
+          echo "Database not ready"
```

Should we put a sleep in the loop, it will "echo" a lot no ?

```suggestion
          echo "Database not ready"
          sleep 2
```